### PR TITLE
DX-492 use larger runner for mypy

### PR DESCRIPTION
**Pull request by jimmylai**

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ on:
 
 jobs:
   run_mypy:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-4-cores
```
Comment (EricVogl-carta) [inline .github/workflows/linters.yml:59]: How does this affect run time and cost?

Comment (EricVogl-carta) [reply .github/workflows/linters.yml:59]: https://carta1.atlassian.net/browse/DX-492?focusedCommentId=308593

If I'm reading that right, it's not too much faster?

**Comment by shawnam** [comment]
@jimmylai is this still a WIP?

**Comment by jimmylai** [comment]
> @jimmylai is this still a WIP?

It's for an experiment and we decided not to proceed with this change based on the result. Closing now.

**Pull request closed without merging**